_posts/2020-10-12-sparklyr-flint-0.2.0-released/sparklyr-flint-0.2.0.Rmd

```diff
@@ -0,0 +1,143 @@
+---
+title: "sparklyr.flint 0.2: ASOF Joins, OLS Regression, and Additional Summarizers"
+description: |
+  We are excited to report a number of powerful new functionalities and improvements which are now part of sparklyr.flint 0.2!
+author:
+  - name: Yitao Li
+    url: https://github.com/yitao-li
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com
+slug: sparklyr.flint-0.2.0
+date: 10-12-2020
+categories:
+  - R
+  - Packages/Releases
+  - Time Series
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/sparklyr-flint-0.2.jpg
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+After [`sparklyr.flint`](https://cran.r-project.org/web/packages/sparklyr.flint/index.html), a [`sparklyr`](https://sparklyr.ai) extension for leveraging [`Flint`](https://github.com/twosigma/flint) time series functionalities through `sparklyr`, has been [introduced](https://blogs.rstudio.com/ai/posts/2020-09-07-sparklyr-flint) in September, we have made a number of enhancements to it, and have successfully submitted `sparklyr.flint` 0.2 to CRAN recently.
+
+In this blog post, we shall highlight the following new features and improvements from `sparklyr.flint` 0.2:
+
+ * [ASOF Joins](#asof-joins) of Timeseries RDDs
+ * [OLS Regression](#ols-regression)
+ * [Additional Summarizers](#additional-summarizers)
+ * [Better Integration With `sparklyr`](#better-integration-with-sparklyr)
+
+## ASOF Joins
+
+For those unfamiliar with the term, ASOF joins are simply temporal join operations based on inexact matching of timestamps. Within the context of [Apache Spark](https://spark.apache.org), a join operation, loosely speaking, matches records from two dataframes (let's call them `left` and `right`) based on some criteria. A temporal join implies matching records in `left` and `right` based on timestamps, and with inexact matching of timestamps permitted, it is typically useful to join `left` and `right` along one of the following temporal directions:
+
+ 1. Looking behind: if a record from `left` has timestamp `t`, then it gets matched with ones from `right` having the most recent timestamp less than or equal to `t`
+ 2. Looking ahead: if a record from `left` has timestamp `t` then it gets matched with ones from `right` having the smallest timestamp greater than or equal to (or alternatively, strictly greater than) `t`
+
+Of course, oftentimes it is not useful to consider two timestamps as "matching" if they are too far apart, therefore an additional constraint on the maximum amount of time to look behind or look ahead is usually also part of a ASOF join operation.
+
+In `sparklyr.flint` 0.2, all ASOF join functionalities of Flint are accessible via the `asof_join()` method. For example, given 2 timeseries RDDs `left` and `right`:
```

Comment:
Here you don't have Flint in backticks, so, either way, just make it consistent.